README.md

```diff
@@ -28,7 +24,23 @@ First, install
 - OpenJPEG [here](https://www.openjpeg.org/), using
 `conda install -c conda-forge openjpeg>=2.3.0`.
 
-Then, create and activate the conda environment:
+Git
+---
+
+Create the virtual environment
+
+    conda env create -f requirements.conda.yml
```

Comment:
Can we add "use requirements.win64.conda.yml if you are working on windows" in readme?

Reply:
Added in this commit https://github.com/TIA-Lab/tiatoolbox/commit/4eaf2f826683f1176d2aca8471fc2dfa3383b3b1